### DISCUSSION TITLE
Current Config File Location

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [jesseduffield/lazygit#2464] [Q&A] [answered]
I seem to have difficulties locating the exact config file for lazygit
I tried 
 - `$env:APPDATA\lazygit\config.yml` 
 - `~\.config\lazygit\config.yml` 
I entered the same customcommand for commitizen but none of this is working as it seems.
the debug log also does not show where a config is loaded.
commit=02bf6a5c177c145c46e5e62524458fc3375d01af, build date=2021-11-22T10:08:20Z, build source=binaryRelease, version=0.31.4, os=windows, arch=amd64

**Comment by mark2185** [comment, marked as answer]
What does running `lazygit --print-config-dir` return?

Could you post your current `config.yml`, maybe it contains an error?

**Reply by sschneider-ihre-pvs** [reply]
\.config\lazygit
```
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: .\.config\lazygit\config.yml
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ customCommand:
   2   │   - key: "C"
   3   │     command: "git cz c"
   4   │     context: "files"
   5   │     loadingText: "opening commitizien commit tool"
   6   │     subprocess: true
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```

**Reply by mark2185** [reply]
I think that should be `customCommands` (with `s` at the end)

**Reply by sschneider-ihre-pvs** [reply]
dooh
the small things in life :D
